convert2rhel_insights_tasks/main.py

```diff
@@ -928,8 +925,20 @@ def main():
             report=str(exception),
         )
     finally:
+        # Report file could be either the pre-conversion or post-conversion
+        # depending on the SCRIPT_TYPE and conversion status. For example:
+        #   - If the SCRIPT_TYPE is analysis, then we will always use pre-conversion report
+        #   - If the SCRIPT_TYPE is conversion, we gonna check the following:
+        #       - If the conversion was successful, use the post-conversion report
+        #       - Otherwise, we probably have the pre-conversion report, so let's use that.
+        report_file = (
+            C2R_POST_REPORT_FILE
+            if IS_ANALYSIS and not os.path.exists(C2R_PRE_REPORT_FILE)
```

Comment:
```suggestion
            if IS_CONVERSION and not os.path.exists(C2R_PRE_REPORT_FILE)
```
We shall not look expect the post-conversion report when doing an analysis.

Reply:
updated, thanks!